auto_install/install.sh

```diff
@@ -296,6 +306,32 @@ distroCheck(){
 				;;
 			esac
 		;;
+		Alpine)
+		PKG_MANAGER='apk'
+		UPDATE_PKG_CACHE="${PKG_MANAGER} update; ${PKG_MANAGER} upgrade --prune"
+		PKG_INSTALL="${PKG_MANAGER} --no-cache add"
+		PKG_COUNT="${PKG_MANAGER} --no-cache upgrade -s | grep -sEe ' [0-9]+ packages' | sed -Ee 's/.* ([0-9]+) packages/\1/' || true"
+		CHECK_PKG_INSTALLED="${PKG_MANAGER} --no-cache info -e"
+
+		# install grepcidr manually
+		## install dependencies
+		"${PKG_INSTALL}" build-base make
+
+		## download binaeries
+		curl -fLo grepcidr.tar.gz --no-cache http://www.pc-tools.net/files/unix/grepcidr-2.0.tar.gz
```

Comment:
Here is a mirror: https://github.com/frohoff/grepcidr
SHA256 of all relevant files match. However, it's a decision whether the original plain HTTP source is more trusted than a non-official HTTPS mirror.
```suggestion
		curl -fLo grepcidr.tar.gz --no-cache https://github.com/frohoff/grepcidr/archive/refs/heads/master.tar.gz
```

Reply:
Having to put a decison on this, this is my decision: 

Get from https mirror, compare the sha256 with the original source. 
If SHA256 fails, Installation should fail, and point towards github for an issue report
If SHA256 Matches, then carry on.

Reply:
> Get from https mirror, compare the sha256 with the original source.

It is sadly not possible to compare the whole archive, since the GitHub mirror contains e.g. an additional `.gitignore`. Only the extracted `grepcidr.c` and other relevant files could be compared.

Reply:
well .. then here it is: https://github.com/pivpn/grepcidr/

https://github.com/pivpn/grepcidr/archive/master.tar.gz

Reply:
Perfect